### PR TITLE
fix: jest config

### PR DESCRIPTION
**Pull request by monty2610**
This PR contain jest config fix to match all test files and added flow plugin to strip flow types

### CODE REVIEW

File: templates/base/config/jest.config.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    rootDir: '../',
+    moduleDirectories: ['node_modules'],
+    testPathIgnorePatterns: ['<rootDir>/dist', '<rootDir>/out/'],
+    clearMocks: true,
+    collectCoverage: true,
```
Comment (ankurkakroo2) [inline templates/base/config/jest.config.js:6]: lets keep coverage thresholds configured as well. Even though they will be set low

File: templates/base/babel.config.js [hunk 2]
```diff
@@ -2,5 +2,6 @@ module.exports = {
   presets: ['@babel/preset-env', '@babel/preset-react'],
   plugins: [
```
Comment (ankurkakroo2) [inline templates/base/babel.config.js:3]: We need to setup environment specific babel configs :)

Comment (vinodloha) [reply templates/base/babel.config.js:3]: @ankurkakroo2 I believe that will come from our NPM module. We can fix it here too for time being

File: templates/base/config/jest.config.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+module.exports = {
```
Comment (vinodloha) [inline templates/base/config/jest.config.js:1]: Can we move the templates/base/config/jest.config.js to templates/base/config/jest/jest.config.js

Comment (monty2610) [reply templates/base/config/jest.config.js]: Done

File: templates/base/config/jest.config.js (hunk 1 above)
Comment (monty2610) [reply templates/base/config/jest.config.js]: done, added a base threshold config

File: templates/base/babel.config.js (hunk 2 above)
Comment (monty2610) [reply templates/base/babel.config.js]: done, moved transform-flow-strip-types to test env

**Pull request merged**